README.md

```diff
@@ -128,102 +134,49 @@ docker run --rm --runtime=nvidia --gpus all --env PATH="${PATH}:/var/lib/snapd/h
 
 ## Development
 
-Developing the `docker` snap package is typically performed on a "classic" Ubuntu distribution.  The instructions here are written for Ubuntu 16.04 "Xenial".
+Developing the `docker` snap package is typically performed on a "classic" Ubuntu distribution (Ubuntu Server / Desktop).
 
-* Install the snap tooling (requires `snapd>2.21` and `snapcraft>=2.26`):
+Install the snap tooling:
 
 ```shell
-sudo apt-get install snapd snapcraft
-sudo snap install core
+sudo snap install snapcraft --classic
 ```
 
-* Checkout this repository and build the `docker` snap package:
+Checkout and enter this repository:
 
 ```shell
-git clone https://github.com/docker/docker-snap
+git clone https://github.com/canonical/docker-snap
 cd docker-snap
-sudo snapcraft
 ```
 
-* Install the newly-created snap package:
+Build the snap:
 
 ```shell
-sudo snap install --dangerous docker_[VER]_[ARCH].snap
+snapcraft -v
 ```
 
-* Manually connect the relevant plugs and slots which are not auto-connected:
+Install the newly-created snap package:
+
+```shell
+sudo snap install --dangerous ./docker_[VER]_[ARCH].snap
+```
+
+Manually connect the relevant plugs and slots which are not auto-connected:
 
 ```shell
 sudo snap connect docker:privileged :docker-support
 sudo snap connect docker:support :docker-support
 sudo snap connect docker:firewall-control :firewall-control
+sudo snap connect docker:network-control :network-control
 sudo snap connect docker:docker-cli docker:docker-daemon
+
 sudo snap disable docker
 sudo snap enable docker
 ```
 
```

Comment:
Here it would be good to add a command to the user compare if it connected all interfaces as it should:

```suggestion
$  snap connections docker                            
Interface         Plug                     Slot                                 Notes
content           -                        docker:config-ro                     -
content           -                        docker:docker-executables            -
content           -                        docker:docker-registry-certificates  -
content           docker:graphics-core22   -                                    -
docker            docker:docker-cli        docker:docker-daemon                 manual
docker-support    docker:privileged        :docker-support                      manual
docker-support    docker:support           :docker-support                      manual
firewall-control  docker:firewall-control  :firewall-control                    manual
home              docker:home              -                                    -
log-observe       docker:log-observe       -                                    -
network           docker:network           :network                             -
network-bind      docker:network-bind      :network-bind                        -
network-control   docker:network-control   :network-control                     manual
opengl            docker:opengl            :opengl                              -
removable-media   docker:removable-media   -                                    -

```

Reply:
I had just removed this. Why would a user want to check this right after performing the connections? What can go wrong that leads to a missing connection? Should the user also check if the snap is installed right after installing it? :) And can a user easily compare such a large output?

Reply:
It has some advantages on leaving this information:
1 - For propagating snapping knowledge: installation is a different (and well known) concept, rather than **connecting interfaces**.
2 - To be able to link to external documentation on snaps, so the user can learn more about snaps concepts like interfaces, slots and plugs.
3 - Some of the interfaces aren't connected, Some of them are auto-connected, so there is a difference that is clear on the output.
4 - The user isn't aware of all interfaces that the docker container had, it's a way of made the user aware of those. 
5 - You end up indirectly teaching the user about a new command `snap connections`.

Reply:
I agree with the above in the context of usage instructions if written in details and not just a dump output of connections. This section is about developing the snap, and people developing the docker snap should already possess advanced snapping knowledge. 

Regarding links to documentation, it can be done without showing the list of connections.

I will add some information about connections in the usage instructions.

Reply:
Added: https://github.com/canonical/docker-snap/pull/168/commits/e6c568a5d9f896debf859e46eecdb822728f55b9